webapi/src/main/scala/org/knora/webapi/responders/v1/OntologyResponderV1.scala

```diff
@@ -412,54 +405,55 @@ final case class OntologyResponderV1Live(
-                                    )
-                                  )
-
-      projectsMap: Map[IRI, ProjectADM] = projectsResponse.projects.map { project =>
-                                            project.id -> project
-                                          }.toMap
-
-      namedGraphs: Seq[NamedGraphV1] = readOntologyMetadataV2.ontologies.toVector
-                                         .map(_.toOntologySchema(InternalSchema))
-                                         .filter { ontologyMetadata =>
-                                           // In V1, the only built-in ontology we show is knora-base.
-                                           val ontologyLabel = ontologyMetadata.ontologyIri.getOntologyName
-                                           ontologyLabel == OntologyConstants.KnoraBase.KnoraBaseOntologyLabel || !OntologyConstants.BuiltInOntologyLabels
-                                             .contains(ontologyLabel)
-                                         }
-                                         .map { ontologyMetadata =>
-                                           val project = projectsMap(ontologyMetadata.projectIri.get.toString)
-
-                                           NamedGraphV1(
-                                             id = ontologyMetadata.ontologyIri.toString,
-                                             shortname = project.shortname,
-                                             longname = project.longname
-                                               .getOrElse(
-                                                 throw InconsistentRepositoryDataException(
-                                                   s"Project ${project.id} has no longname"
-                                                 )
-                                               ),
-                                             description = project.description.headOption
-                                               .getOrElse(
-                                                 throw InconsistentRepositoryDataException(
-                                                   s"Project ${project.id} has no description"
-                                                 )
-                                               )
-                                               .toString,
-                                             project_id = project.id,
-                                             uri = ontologyMetadata.ontologyIri.toString,
-                                             active = project.status
-                                           )
-                                         }
-
-      response = NamedGraphsResponseV1(
-                   vocabularies = namedGraphs
-                 )
+      readOntologyMetadataV2 <-
+        messageRelay
+          .ask[ReadOntologyMetadataV2](
+            OntologyMetadataGetByProjectRequestV2(
+              projectIris = projectIris.map(_.toSmartIri),
```

Comment:
The `_.toSmartIri` needs an implicit `StringFormatter` and throws.
Should be replaced with `IriConverter.asSmartIri`.

Reply:
I added new method which solves IRI conversion for multiple IRIs.